Cloud Application Manager/Tutorials/publish-script-box.md

```diff
@@ -20,7 +20,17 @@ Click on the gear next to the version of your application that you would like to
 
 ![public-box-publish.png](../../images/cloud-application-manager/public-box-publish.png)
 
-Fill out the "Request to Publish Box" form. The "Publisher" field should be either your name, your team's name, or your company name. The "Category" field will specify what section of the catalog your box will be displayed under. Add any additional comments that you would like to be submitted to the Cloud Application Manager administrators, check the "I agree" box and hit send.
+Fill out the "Request to Publish Box" form. The "Publisher" field should be either your name, your team's name, or your company name. 
+
+The "Visibility Type" field is present to specify wether the box should be published as:
+
+* *Open*: box will be made public as open-source
+* *Execute*: box will be made public as execute-only, so it will only be deployable and only public variables will be visible.
+* *Closed*: box will be used to embed into other boxes as box variables, but event code will be hidden. Only public variables will be visible. 
```

Comment:
```suggestion
* *Closed*: box will be published with its event source code as hidden. Only public variables will be visible. The box can be deployed directly or it can be embedded into another box as a box variable to be deployed as part of the enclosing box deployment.
```